dynawaltz/src/main/java/com/powsybl/dynawaltz/models/buses/StandardBus.java

```
@@ -90,4 +90,13 @@ public String getSwitchOffSignalVarName() {
     public String getNumCCVarName() {
         return "@NAME@_numcc";
     }
+
+    @Override
+    public String getUImpinVarName() {
+        return "@NAME@_U";
+    }
+
+    public String getUpuImpinVarName() {
+        return "@NAME@_Upu";
+    }
```

These var names do not exist for this model, we should either throw an exception or return Optional.empty()